### DISCUSSION TITLE
How to allow Renovate bypass Pull Request review on protected branch?

### DISCUSSION THREAD

**Discussion by tal-rofe** [renovatebot/renovate#20309] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I want to allow Renovate to make PR without review from CODEOWNERS.

I have a protected branch, `main` on which each Pull request to it, must have review from 1, and from `CODEOWNERS`.
I tried to allow Renovate skip it, but it still requires me to review it and Renovate won't merge it automatically until I approve the pull request manually.

My Renovate configuration allows any `patch`, `minor` PRs to be automatically merged:
```json
	"packageRules": [
		{
			"updateTypes": ["minor", "patch"],
			"automerge": true
		},
```

And this is how I've configured the protected `main` branch:
<img width="822" alt="Screenshot 2023-02-09 at 13 07 43" src="https://user-images.githubusercontent.com/100444463/217796135-a4b4123a-deaf-41e9-8636-291b20a78e1a.png">
<img width="801" alt="Screenshot 2023-02-09 at 13 07 56" src="https://user-images.githubusercontent.com/100444463/217796189-63ab45cf-7997-4146-81e1-9773efdd1c70.png">

### What you tried so far.

I tried to allow Renovate to bypass Review , you can see my configurations.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment]
use the renovate auto approve bot. 😉

https://github.com/apps/renovate-approve

**Reply by rarkins** [reply]
I'm not sure that works because bots can't be code owners

**Comment by rarkins** [comment]
To my knowledge there's no way to bypass GitHub code owner restrictions
